subprojects/base-services/src/test/groovy/org/gradle/internal/service/ServiceLocatorTest.groovy

```diff
@@ -115,7 +115,7 @@ org.gradle.ImplClass2""")
 
         then:
         RuntimeException e = thrown()
-        e.message == "Could not determine implementation class for service 'java.lang.String' specified in resource '${serviceFile}'."
+        e.message == "Could not determine implementation class for service 'java.lang.String' specified in resource '" + serviceFile + "'."
```

Comment:
Why not use GStrings any more? Did something change in Groovy about equality?

Reply:
Yeah, a null value doesn't seem to evaluate to `null` but to an empty string now.